crunchy-xml-decoder/hls.py

```diff
@@ -71,7 +71,7 @@ def copy_with_decrypt(input, output, key):
 def fetch_streams(output, video):
     output = open(output, 'wb')
     for n, seg in enumerate(video.segments):
-        sys.stdout.write('\x1b[2K\r%d/%d' % (n + 1, len(video.segments)))
+        sys.stdout.write('\r%d/%d' % (n + 1, len(video.segments)))
```

Comment:
Yeah, I see your point. But still, I hardly can see any code in this file where we're writing additional data to stdout, and on top of that, ANSI sequences aren't exactly supported on all Windows terminals, not until W10 at least, which makes it even weirder since the reported screenshot seems to be on W10. But I dunno, we could use ``coloroma`` to support escape sequences in a cross-platform manner, if we really must use ANSI here, but I'm not sure if it worth adding another dependency for basically a simple progress reporting functionality. I'm open to suggestions.